pkg/kstatus/polling/polling.go

```diff
@@ -56,6 +67,8 @@ type Options struct {
 	// all needed resources before each polling cycle. If this is set to false,
 	// then each resource will be fetched when needed with GET calls.
 	UseCache bool
+
+	CustomStatusReadersFactoryFunc func(engine.ClusterReader, meta.RESTMapper) map[schema.GroupKind]engine.StatusReader
```

Comment:
Can we add a comment to this field explaining what it does?

Reply:
done